### PR TITLE
DOC: Fix PEP-8 issues in 10min.rst

### PR DESCRIPTION
**Pull request by FHaase**

### CODE REVIEW

File: doc/source/10min.rst [hunk 1]
```diff
@@ -31,9 +14,22 @@ Customarily, we import as follows:
 
 .. ipython:: python
 
-   import pandas as pd
    import numpy as np
-   import matplotlib.pyplot as plt
+   import pandas as pd
+
+.. ipython:: python
+   :suppress:
```
Comment (datapythonista) [inline doc/source/10min.rst]: why not at the beginning and with numpy and pandas imports in the same suppress block? I think this is what we do in all the other pages, right?

File: doc/source/10min.rst [hunk 2]
```diff
@@ -718,8 +720,10 @@ of the columns with labels:
                      columns=['A', 'B', 'C', 'D'])
    df = df.cumsum()
 
+   plt.figure()
+   df.plot()
    @savefig frame_plot_basic.png
-   plt.figure(); df.plot(); plt.legend(loc='best')
+   plt.legend(loc='best')
```
[the remark this answers is not in the document]
Comment (FHaase) [reply doc/source/10min.rst]: It's basically mixed sometimes with sometimes without.
I'd prefer to put `;` at the end to suppress the output, but you've asked to PR that in seperate PR's and fixing PEP-8 issues is to have no `;` at the end of a line.

I'm waiting for #23847 in order to ignore `E703` on a file-based basis.

**Review by datapythonista — APPROVED** [review]
ok, let's merge this then, we can add semicolons to hide the plots output in a separate PR (we can open an issue to discuss, in case someone prefers not to ignore the flake8 rule

**Comment by jreback** [comment]
thanks @FHaase

**Pull request merged**